DCO

```diff
@@ -0,0 +1,37 @@
+Developer Certificate of Origin
+Version 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
```

Comment:
You should put Algoo, here... no?
by the way, copyright should be 2013, 2018

Reply:
I don't think we should put it there, DCO v1.1 copyright owner are Linux Foundation.
Gitlab itself, let linux foundation copyright : https://gitlab.com/gitlab-org/dco/blob/master/README.md .

But you're right for date in MIT license.

Reply:
ok, that's fine